frontend/selection.tsx

```diff
@@ -0,0 +1,14 @@
+import { Data } from "./data";
+import { Rect } from "./rect";
+
+export function Selection({ data, shapeID }: { data: Data; shapeID: string }) {
+  return (
+    <Rect
```

Comment:
This will become fancier when we add drag handles to it, hence being in its own component.